src/js/utils/settings.js

```diff
@@ -0,0 +1,17 @@
+import { filter, fromPairs, map, toPairs } from "lodash"
+
+export const getHostOverridesFromSettings = (settings, removePrefix) => {
+  return fromPairs(
+    map(
+      filter(toPairs(settings), (item) => {
+        return item[0].indexOf("hostOverrides") !== -1
+      }),
+      (item) => {
+        if (removePrefix) {
+          item[0] = item[0].replace("hostOverrides:", "")
+        }
+        return item
+      },
+    ),
+  )
+}
```

Comment:
This is not very readable. Please use the dot-free-notation with utilities from "lodash/fp" or the dot-notation, e.g.

```javascript
import { pipe, filter, fromPairs, map, toPairs } from "lodash/fp"

export const getHostOverridesFromSettings = (settings, removePrefix) =>
  pipe(
    toPairs,
    filter((item) => item[0].indexOf("hostOverrides") !== -1),
    map((item) => {
      if (removePrefix) {
        item[0] = item[0].replace("hostOverrides:", "")
      }
      return item
    }),
    fromPairs,
  )(settings)
```

Reply:
Valid point!